### PR TITLE
Import a stand-alone starter content file

### PR DESCRIPTION
**Pull request by bph**
- Write the content of the file starter-content.php into the plugin directory 
- Active the plugin 
- Use the "importThemeStarterContent" step. 
- User lands on home page and can see the starter content on the "Our Subscriptions" page. 
Screenshot 2024-11-27 at 17 15 37

Thank you to @bgrgicak for inspiration.

### CODE REVIEW

File: GALLERY.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ Here's the list of all the community Blueprints submitted to this repository. Se
...
+| Import a standalone starter content via a blueprint step                                          | BImport a standalone starter content via a blueprint step                                                                                                                                     | [@bph](https://github.com/bph)                             | • [Open in Playground](https://playground.wordpress.net/?blueprint-url=https://raw.githubusercontent.com/wordpress/blueprints/trunk/blueprints/filestartercontent/blueprint.json)<br>• [View source](https://github.com/wordpress/blueprints/blob/trunk/blueprints/filestartercontent/blueprint.json)<br>• [Edit](https://playground.wordpress.net/builder/builder.html?blueprint-url=https://raw.githubusercontent.com/wordpress/blueprints/trunk/blueprints/filestartercontent/blueprint.json)                                                                   |
```
Comment (zaerl) [inline GALLERY.md]: I think we got a typo here. `BImport`

File: blueprints/filestartercontent/blueprint.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
```
Comment (zaerl) [inline blueprints/filestartercontent/blueprint.json]: I see other blueprints directories are in the form `this-that-another`. Maybe we should name this `file-starter-content`? What do you think?

File: blueprints/filestartercontent/startercontent.php [hunk 3]
```diff
@@ -0,0 +1,32 @@
+<?php
+/*
+Plugin Name: Short Starter Content Example
+Description: This plugin adds a starter contentto a Playground instance. Used as part of a Playground Blueprint example.
```
Comment (zaerl) [inline blueprints/filestartercontent/startercontent.php]: Small typo here. "contentto".

**Review by zaerl — CHANGES_REQUESTED** [review]
It needs a couple of fixes to make. Good work.

File: blueprints/filestartercontent/startercontent.php (hunk 3 above)
Comment (bph) [reply blueprints/filestartercontent/startercontent.php]: Good eye. Thank you. Fix

File: blueprints/filestartercontent/blueprint.json (hunk 2 above)
Comment (bph) [reply blueprints/filestartercontent/blueprint.json]: Good idea, keeping consitency.

File: GALLERY.md (hunk 1 above)
Comment (bph) [reply GALLERY.md]: Oops. Thanks. Copy/paste devil. 🤦‍♀️ Shortened the title, too

**Review by zaerl — APPROVED** [review]
Thanks for the fixes. 🚢

**Pull request merged**